### DISCUSSION TITLE
Connection issues to the web server

### DISCUSSION THREAD

**Discussion by blueglyph** [community/community#38772] [Projects and Issues]
### Select Topic Area

Bug

### Body

For the 2 last days,
- browsing issues on github.com is very slow, sometimes the server refuses the connection
- filling in issues in projects fails to correctly switch from Write to Preview mode. When switching back to Write mode after checking the preview, parts of the text are missing; sometimes they reappear, sometimes not
- browsing the code or the commits is very slow and fails now and then

I saw no announcements nor acknowledgements of this problem, the [status](https://www.githubstatus.com/) is clear, yet it there are of GitHub problems here and on several sites such as downdetector.com and similar.

Did I miss announcement about maintenance or server issues?

**Comment by ossanna16** [comment]
Hi there @blueglyph and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by blueglyph** [comment]
Thanks @ossanna16 :)

**Comment by jfullstackdev** [comment]
never had that experience in my entire years using GitHub, even if my past computer was just Celeron and my connection was just hotspot with limited data consumption

maybe you have so many extensions installed, low RAM, slow Internet connection, firewall and antivirus that blocks the site etc.

**Reply by blueglyph** [reply]
> never had that experience in my entire years using GitHub, even if my past computer was just Celeron and my connection was just hotspot with limited data consumption
> 
> maybe you have so many extensions installed, low RAM, slow Internet connection, firewall and antivirus that blocks the site etc.

It has nothing to do with my system or my connection. I never had those issues before, but this week has been hell, while any other website and service was fine. As I mentioned, it was reported by other people too, there were high spikes on downdetector for instance. But indeed that doesn't seem to affect everyone.

It seems to be gradually back to normal, so I suppose that they are working on the problem, whoever "they" are (GitHub or network infrastructure). But it's the weekend, let's hope the downtimes don't come back on Monday.

It doesn't help that GitHub didn't respond to this nor the other similar reports, it makes it harder to narrow down the problem.
